.buildkite/release-pipelines/complete-code-freeze.yml

```
@@ -0,0 +1,17 @@
+env:
+  IMAGE_ID: $IMAGE_ID
+
+# The code freeze completion needs to run on macOS because it uses genstrings under the hood.
+agents:
+    queue: mac
+
+steps:
+  - label: Complete Code Freeze
+    key: complete_code_freeze
+    plugins: [$CI_TOOLKIT_PLUGIN]
+    command: .buildkite/commands/complete-code-freeze.sh $RELEASE_VERSION
```

See https://github.com/Automattic/simplenote-ios/pull/1646/files#r1742990234 for more context about this suggestion:

```suggestion
    command: .buildkite/commands/complete-code-freeze.sh $RELEASE_VERSION
    retry:
      manual:
        # If those jobs fail, one should always prefer re-triggering a new build from ReleaseV2 rather than retrying the individual job from Buildkite
        allowed: false
```

---

3ad44174